docs/source/index.rst

```diff
@@ -62,6 +62,21 @@ termination using the message id.
 
   abort(message_id)
 
+Abort a task using a custom abort_ttl value
+------------
+
+By default, abort has a limited window of 90,000 milliseconds. This means a worker will skip a task only if the task was aborted up to 90 seconds ago. In case of longer delay in the task processing this value can be overridden.
+
+.. code-block::
+
+  @dramatiq.actor
+  def count_words(url): ...
+
+  message = count_words.send_with_options(args=("https://example.com",), delay=60*60*1000)
+  message_id = message.message_id
+
+  abort(message_id, abort_ttl=271000)
```

Comment:
The `abort_ttl` value is still to low to abort the task above. Let's use something like `2*60*60*1000` instead!

Reply:
My bad change added as requested.